tests/rptest/tests/datalake/datalake_e2e_test.py

```diff
@@ -7,23 +7,40 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 from rptest.services.cluster import cluster
-from rptest.tests.datalake.datalake_services import DatalakeServicesBase
+from rptest.services.redpanda import SISettings
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.tests.datalake.datalake_services import DatalakeServices
+from rptest.tests.datalake.query_engine_base import QueryEngineType
+from ducktape.mark import matrix
 
 
-class DatalakeE2ETests(DatalakeServicesBase):
+class DatalakeE2ETests(RedpandaTest):
     def __init__(self, test_ctx, *args, **kwargs):
-        super(DatalakeE2ETests, self).__init__(test_ctx,
-                                               num_brokers=1,
-                                               *args,
-                                               **kwargs)
+        super(DatalakeE2ETests,
+              self).__init__(test_ctx,
+                             num_brokers=1,
+                             si_settings=SISettings(test_context=test_ctx),
+                             extra_rp_conf={
+                                 "iceberg_enabled": "true",
+                                 "iceberg_catalog_commit_interval_ms": 5000
+                             },
+                             *args,
+                             **kwargs)
+        self.test_ctx = test_ctx
         self.topic_name = "test"
 
-    @cluster(num_nodes=5)
-    def test_e2e_basic(self):
+    @cluster(num_nodes=4)
```

Comment:
Huh I would've thought we'd only need 3 nodes:
1. redpanda
2. trino/spark
3. Iceberg REST catalog adapter

Reply:
kgo producer.